src/models/slices/filters.ts

```diff
@@ -1,7 +1,10 @@
-import { CombinedFilter, DisplayableFacet, Filter, SortBy } from '@yext/answers-core';
+import { DisplayableFacet, SortBy } from '@yext/answers-core';
+import { SelectableFilter } from '../utils/selectablefilter';
 
 export interface FiltersState {
-  static?: Filter | CombinedFilter | null;
+  static?: {
+   [filterSetId: string]: SelectableFilter[] | null;
```

Comment:
To make sure I'm understanding this correctly, we are creating a concept of a filterSetId which groups together filters. If we were to have multiple static filter components, they would each use the filters associated with that filterSetId. Is that correct? Also, how will we decide which key to use for AppliedFilters which won't have a filterSetId?

Reply:
That's correct. For each static filter set, headless will store the corresponding id and a list of filters. As for AppliedFilters. It would run through all the sets to find the corresponding filter to unselect.